### ISSUE TITLE
/:params

### ISSUE THREAD

**Issue by SleepyBrett** [ttezel/unio#18]
Having a little trouble using :params in a quickie instagram spec.

I have the following in the spec:

``` json
    {
      "name": "tags/:id",
      "path": "tags/:id",
      "methods": [
        "get"
      ],
      "params": {
        "access_token": "required"
      }
    },
    {
      "name": "tags/:id/media/recent",
      "path": "tags/:id/media/recent",
      "methods": [
        "get"
      ],
      "params": {
        "min_id": "optional",
        "max_id": "optional",
        "access_token": "required"
      }
    },
```

But when I make the following request:

``` javascript
.get('tags/'+tag+'/media/recent', {'access_token':at}, function (err,resp,body) {
```

Unio seems to hit on "tags/:id" and not "tags/:id/media/recent".

Suggestions?

**Comment by SleepyBrett** [comment]
my current workaround is

``` json
{
      "name": "tagsMediaRecent/:id",
      "path": "tags/:id/media/recent",
```

**Comment by ttezel** [comment]
@SleepyBrett Could you paste your whole instagram spec here? It would help debugging.

**Comment by ttezel** [comment]
Hey @SleepyBrett, I have fixed the bug and published unio version `1.1.4` with the fix. Run `npm update unio` to get it. I have added an instagram spec to the specs folder, so you can use that spec directly and make the requests that weren't working for you before. I also added a couple tests for the instagram API in the test folder. You can check out the code there too.

Closing this issue since it's fixed, feel free to reopen if needed.
